developers/blobstream.md

```diff
@@ -14,89 +14,111 @@ on Ethereum, for integration by developers into L2 contracts. This enables Ether
 
-It works by relying on a set of signers to attest to some event on Celestia:
-the Celestia validator set. The Blobstream contract keeps track of the
-Celestia validator set by updating its view of the validator set with
-`updateValidatorSet()`. More than 2/3 of the voting power of the current
-view of the validator set must sign off on new relayed events, submitted with
-`submitDataRootTupleRoot()`. Each event is a batch of `DataRootTuple`s, with
-each tuple representing a single
-[data root (i.e. block header)](https://celestiaorg.github.io/celestia-app/specs/data_structures.html#header).
-Relayed tuples are in the same order as Celestia block headers.
+Canonical deployments of Blobstream X will be maintained on the
+following chains: Arbitrum One, Base and Ethereum Mainnet. Every 1
+hour, Succinct will post an update to the Blobstream X contract
+that will include a new data commitment range that covers a 1-hour
+block range from the `latestBlock` in the Blobstream X contract.
 
-![Blobstream attestation flow](/img/blobstream/Celestia_Blobstream_attestation_flow.jpg)
+:::tip NOTE
+Custom ranges can be requested using the `BlobstreamX` contract
+to create proofs for specific Celestia block batches. These ranges
+can be constructed as `[latestBlock, customTargetBlock)`, with
+`latestBlock` is the latest block height that was committed to by the
+`BlobstreamX` contract, and `latestBlock > customTargetBlock`,
+and `customTargetBlock - latestBlock <= DATA_COMMITMENT_MAX`.
+
+Block ranges that are before the contract's `latestBlock` can't be
+proven a second time in different batches.
+
+More information can be found in [`requestHeaderRange(...)`](https://github.com/succinctlabs/blobstreamx/blob/364d3dc8c8dc9fd44b6f9f049cfb18479e56cec4/contracts/src/BlobstreamX.sol#L78-L101)
+method.
+:::
 
-### Events and messages relayed
+### How Blobstream X works
 
-**Validator sets**:
-The relayer informs the Blobstream contract who are the current
-validators and their power.
-This results in an execution of the `updateValidatorSet` function.
+As shown in the diagram below, the entrypoint for updates to the Blobstream
+X contract is through the `SuccinctGateway` smart contract, which is a
+simple entrypoint contract that verifies proofs (against a deployed
+onchain verifier for the Blobstream X circuit) and then calls the
+`BlobstreamX.sol` contract to update it. More information about the
+`SuccinctGateway` can be found here: [TODO].
 
-**Batches**:
-The relayer informs the Blobstream contract of new data root tuple roots.
-This results in an execution of the `submitDataRootTupleRoot` function.
+![blobstream x overview diagram draft](/img/blobstream/blobstream_x_overview_diagram_draft.png)
 
-## How to integrate
+<!-- markdownlint-disable MD042 -->
 
-Integrating your L2 with Blobstream requires two components: your onchain smart
+:::tip NOTE
+If the contract is still not deployed, then it needs to be
```

Comment:
nit: "If the Blobstream X contract is not deployed on a desired chain, it needs to be deployed before it can be used by your rollup. See ..."

Reply:
[d7d2f14](https://github.com/celestiaorg/docs/pull/1379/commits/d7d2f14e872ee4fcc5a06079a3e5b0c29840e0f2)